jobs/jobs_test_utils.py

```diff
@@ -80,37 +89,30 @@ def tearDown(self):
         self.model_io_stub.clear()
         super(JobTestBase, self).tearDown()
 
-    def run_job(self, pipeline=None, runner=None, options=None):
+    def run_job(self):
         """Runs a new instance of self.JOB_CLASS and returns its output.
 
-        Args:
-            pipeline: type(beam.Pipeline). The type of pipeline that will be
-                used to manage the job. Uses TestPipeline is not provided.
-            runner: type(PipelineRunner). The type of runner that will be used
-                to execute the pipeline. Uses DirectRunner if not provided.
-            options: JobOptions|None. A JobOptions instance used to configure
-                the pipeline and job. Uses options with runtime type checks and
-                stubbed model IO by default.
+        Test authors should override this method if their jobs need special
```

Comment:
Could you clarify what "need special inputs" means, perhaps with a concrete example?